server/src/main/java/com/cloud/storage/snapshot/SnapshotManagerImpl.java

```diff
@@ -1327,32 +1322,11 @@ private void updateSnapshotPayload(long storagePoolId, CreateSnapshotPayload pay
     }
 
     private DataStoreRole getDataStoreRole(Snapshot snapshot, SnapshotDataStoreDao snapshotStoreDao, DataStoreManager dataStoreMgr) {
```

Comment:
The `dataStoreMgr` will not be used, it would be better to remove it from the method signature.